website/docs/tutorials/08-cairo-1-support/02-project-initialization.md

```diff
@@ -14,100 +16,285 @@ protostar init-cairo1 my-project
 After running the command, the following structure will be generated:
 
 ```
-my-project/
-├── src/
-│   └── main.cairo
-│   └── lib.cairo
+my_project/
+├── hello_starknet/
+│   ├── src/
+│   │   ├── business_logic/
+│   │   │   └── utils.cairo
+│   │   ├── contracts/
+│   │   │   └── hello_starknet.cairo
+│   │   ├── business_logic.cairo
+│   │   ├── contracts.cairo
+│   │   └── lib.cairo
 │   └── cairo_project.toml
 ├── tests/
-│   └── test_main.cairo
+│   ├── test_hello_starknet.cairo
+│   └── test_utils.cairo
 └── protostar.toml
 ```
 
-This template contains:
-
-- `src` directory
-    - `cairo_project.toml` which is needed for compilation 
-    - `main.cairo` file with one function definition
-    - `lib.cairo` file which defines the module
-- `test` directory
-    - Single test file with one test for function defined in `main.cairo`
-- `protostar.toml` containing information about the project
-
 :::warning
 This template will be changed in future versions, but the old one will still be usable with newer protostar versions
 :::
 
+### `hello_starknet`
+
+This directory contains our only crate in this project - `hello_starknet`.
```

Comment:
```suggestion
This directory contains our only module in this project - `hello_starknet`.
```

Reply:
This is confusing, we don't know a difference between those (is there any difference?)